### DISCUSSION TITLE
mjs file can't import PrismaClient

### DISCUSSION THREAD

**Discussion by jet10000** [prisma/prisma#4899] [Q&A]
```js
import { PrismaClient } from "@prisma/client"

const prisma = new PrismaClient()
```

test.mjs

**Comment by ryands17** [comment]
Hey @jet10000 👋 

Unfortunately Prisma doesn't support `.mjs` currently but there's work in progress for supporting `.mjs` files and you can follow this request [here](https://github.com/prisma/prisma-client-js/issues/539) for more details :)

**Comment by JakubKoralewski** [comment]
I think you may be trying to create a seeding script, is that right? I also had problems with getting TypeScript to work as a script. Using `tsconfig-paths` was what fixed it for me:
```
"scripts": {
  "dev-seed": "ts-node -r tsconfig-paths/register -P prisma/seeding/tsconfig.script.json prisma/seeding/initiateDb.ts"
},
```
You can see the contents of my seeding script [here](https://github.com/JakubKoralewski/lets-watch-it-together/blob/da94556af94fcd0c36c95951864225808a29770d/prisma/seeding/initiateDb.ts). Sorry if that's not what you were looking for.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
